services/common/src/setupTests.ts

```diff
@@ -23,7 +31,7 @@ jest.mock("@mds/common/providers/featureFlags/useFeatureFlag", () => ({
     isFeatureEnabled: () => true,
   }),
 }));
-
+window.scrollTo = jest.fn();
```

Comment:
I was getting so many console errors from window.scrollTo and useLayoutEffect that I ran out of terminal lines and couldn't see the tests that were actually failing! Mocking made that better.